redis-snapshot-replicator/main.tf

```diff
@@ -241,8 +231,8 @@ resource "aws_lambda_function" "redis_create_snapshot" {
   role          = aws_iam_role.iam_for_lambda_redis[0].arn
   handler       = "create_snapshot.lambda_handler"
 
-  filename         = local.filename
-  source_code_hash = filebase64sha256(local.filename)
+  filename         = "${path.module}/shipper.zip"
+  source_code_hash = filebase64sha256("${path.module}/shipper.zip")
```

Comment:
The `archive_file` data source already outputs the base64 encoded sha256 of the archive: https://www.terraform.io/docs/providers/archive/d/archive_file.html#output_base64sha256

So you don't need to re-compute it here.

Reply:
nice! didn't know that. Going to use it